build.gradle

```diff
@@ -89,7 +89,8 @@ nugetSpec {
             summary: 'ILRepack is a utility that can be used to merge multiple .NET assemblies into a single assembly'
         ],
         files: [
-            { file(src: repack.ext.repacked, target: 'tools') }
+            { file(src: repack.ext.repacked, target: 'tools') },
+            { file(src: "..\\..\\..\\ILRepack.props", target: 'build') }
```

Comment:
Can probably use `project.file("ILRepack.props")` to have a cleaner (absolute) path, no?